config/locales/time/en.yml

```diff
@@ -2,6 +2,7 @@
 en:
   date:
     day_names:
+      -
```

Comment:
adding this line allows `%A` to grab the correct day of the week, versus previously when it was giving the name of the next day

Reply:
should we add that as a YAML comment?
```suggestion
      - # empty item for correct weekday offset for %A
```